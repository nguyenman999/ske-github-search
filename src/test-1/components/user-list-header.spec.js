import React from 'react'
import { shallow } from 'enzyme'
import UserListHeader from './user-list-header'

describe('user-list-header', () => {
  let props
  beforeEach(() => {
    props = {
      title: 'title',
      onChange: jest.fn(),
      onSearch: jest.fn(),
    }
  })

  it('render: default', () => {
    const wrapper = shallow(<UserListHeader {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
