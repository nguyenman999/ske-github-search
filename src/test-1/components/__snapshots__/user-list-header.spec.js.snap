// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user-list-header render: default 1`] = `
<Alignment
  className=""
>
  <Alignment.Left
    className=""
  >
    <Title
      level={4}
    >
      title
    </Title>
  </Alignment.Left>
  <Alignment.Right
    className=""
  >
    <Search
      enterButton={false}
      onChange={[MockFunction]}
      onSearch={[MockFunction]}
      placeholder="Input search text"
    />
  </Alignment.Right>
</Alignment>
`;
